### PR TITLE
Past-project-review

### PR DESCRIPTION
**Pull request by VTY1999**
**These are changes we have implemented for this project**

- Creat a new repository
- creat a new branch
- Add descriptive README file
- Creat a linters for html and css

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @VTY1999, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

## You have submitted a project, that does not have at least two review comments from you. Please consider reading your project requirements and following the instructions on how to perform a code review on your past project and submit it for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <p>Copyright 2016, Original Trombones</p>
+    
+</footer>
+    
+
+
+
+
+
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (VTY1999) [inline index.html]: **Hi VTY1999**
Great work so far, but you need to remove the script tag.

**Review by VTY1999 — COMMENTED** [review]
**Great job**
You have a clean code and your style is on point. 
There is nothing else to say happy coding

File: index.html (hunk 1 above)
Comment (Cathella) [reply index.html]: Hi @VTY1999, you have done a great job so far reviewing your code. 
> This project is about the you performing a code review on this project by checking if you applied HTML & CSS best practices

I kindly suggest that you look through all of your code and find anywhere you might have not passed the standards, or a part of code you would refactor better 😄.

Start by "quoting the code", then state the aimed problem ❗ , politely offer a suggestion 😏 and conclude with a reason or importance of making that change.

Please follow the above guideline, ☝🏻 and make the review comment you made better! Also don't not forget to add as many reviews to your code as you can (the minimum is 2).

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Requested 🛑

### Hi @VTY1999 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To highlight 🍾

✔️ Submitted review in the Pull Request.
✔️ Utilized the "Review changes" feature from GitHub.
✔️ Used the GitHub inline comments feature


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ]  Kindly, I would suggest you use a gif or emojis in your review description to make the review look friendly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+.articles {
+  display: flex;
+  flex-wrap: wrap;
+  width: 50%;
+  padding: 0;
+  margin: auto;
+}
+
+.articles .side-band {
+  display: flex;
```
Comment (VTY1999) [inline style.css]: **Hi VTY1999**
Great work so far, You used display: flax in the side band but i would have preferred it if you us float: left so it is more organized.
```suggestion
  float: left;
```

**Review by VTY1999 — COMMENTED** [review]
**Hi VTY1999**
Great job, you have competed the changes. There is nothing else to say.

1Sfs
happy coding.

**Review by HENRYKC24 — APPROVED** [review]
# STATUS: APPROVED  :heavy_check_mark: 
## Hi @VTY1999 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**